index.html

```diff
@@ -185,6 +279,431 @@ <h4 class="money-funded">$530M</h4>
+            </section>
+            <ul class="categories-list">
+                <li>
+                    <a class="list-link" href="#">Twitter shares fall 7% in pre-market after Tump's removal</a>
+                </li>
+                <li>
+                    <a class="list-link" href="#">Winklevoss comes out of meditation to witness Bitcoin's record high</a>
+                </li>
+                <li>
+                    <a class="list-link" href="#">CD Project Rekt: Cyberpunk 2077 tanks studio's share price nearly 50%</a>
+                </li>
+                <li>
+                    <a class="list-link" href="#">CD Project stock down to 20% after Cyberpunk reviews hit the internet</a>
+                </li>
+                <li>
+                    <a class="list-link" href="#">AirBnb to be bigger than eBay after its long awaited IPO</a>
+                </li>
+             </ul>
+
+        </section>
+
+        <section class="distract">
+            <a href="#" class="fund-heading"><h2 class="funding-header">Distract &gt;</h2></a>
+            <section class="distract-most-popular">
+                <h4 class="pre-title">MOST POPULAR</h4>
+                <h3 class="big-title">GameStop stocks reach record high -- thanks to Redditors</h3>
+                <div class="cat-details">
+                    <p><a class="cat-author" href="#">Ivan Mehta</a> • 1 day ago</p>
+                </div>
+            </section>
+            <section class="categories-latest">
+                <h4 class="categories-pre-title">Latest</h4>
+                <p class="categories-big-title">Alibaba shares jump 8% as Jack Ma surfaces after 3 months</p>
+                <img class="categories-thumbnail" alt="thumbnail" src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2021/01/bloodlines.jpg">
+            </section>
+            <ul class="categories-list">
+                <li>
+                    <a class="list-link" href="#">Twitter shares fall 7% in pre-market after Tump's removal</a>
+                </li>
+                <li>
+                    <a class="list-link" href="#">Winklevoss comes out of meditation to witness Bitcoin's record high</a>
+                </li>
+                <li>
+                    <a class="list-link" href="#">CD Project Rekt: Cyberpunk 2077 tanks studio's share price nearly 50%</a>
+                </li>
+                <li>
+                    <a class="list-link" href="#">CD Project stock down to 20% after Cyberpunk reviews hit the internet</a>
+                </li>
+                <li>
+                    <a class="list-link" href="#">AirBnb to be bigger than eBay after its long awaited IPO</a>
+                </li>
+             </ul>
+
+        </section>
+
```

Comment:
*section* tags wrapped in the category section should be changed to *article* tags